docs/operations/BACKLOG_WEBHOOK_SECRET.md

```
@@ -0,0 +1,76 @@
+
+# production
+cd backend && npx wrangler secret put BACKLOG_WEBHOOK_SECRET --env production
+```
+
+プロンプトが表示されたら、手順1で生成した値を貼り付ける。
+
+### 3. ローカル開発環境への設定
+
+`backend/.dev.vars` に追加:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)